src/pages/account/AdminPage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import styled from "styled-components";
+
+import axiosApi from "../../utils/AxiosApi";
+import Card from "../../components/Card";
+import NavBar from "../../components/NavBar";
+import Button from "../../components/Button";
+
+const AdminPage = () => {
+  const [stories, setStories] = useState([]);
+  const [child, setChild] = useState({});
+  const [id, setId] = useState("");
+  const { childId } = useParams();
+
+  useEffect(() => {
+    const childData = async () => {
+      const response = await axiosApi(`/getChild/${childId}`);
+      const childData = await response.data;
+      const storyData = await response.data.stories;
+      setChild(childData);
+      setStories(storyData);
+    };
+    return childData();
+  }, [childId]);
+
+  const deleteStory = async (event) => {
+    event.preventDefault();
+    if (id) {
+      await axiosApi.post(`deleteStory/${id}`, { childId });
```

Comment:
Shouldn't you have some sort of response to check if it deleted the post correctly?

Reply:
With the response you could check for example the status code if it was deleted successfully.
If the status code is good, you can `get` the child data.

e.g:

```javascript
const deleteStory = await axiosApi.post(`deleteStory/${id}`, { childId });
const responseDelete = deleteStory.data;

if (responseDelete.status === 200) {
  const getChildData = await axiosApi(`/getChild/${childId}`);
  const storyData = await response.data.stories;
  setStories(storyData);
}
```

Like this you only get the child data **IF**  the data gets deleted successfully.
Your current setup is _always_ fetching the child data regardlessly

Reply:
Obviously, if something goes wrong you should also have it in an error message so also wrap it around a `try/catch` block. Especially because you're doing multiple requests